src/norm/sqlite.nim

```diff
@@ -352,10 +352,11 @@ template transaction*(dbConn; body: untyped): untyped =
     raise
 
 proc selectOneToMany*[O: Model, M: Model](dbConn; oneEntry: O, relatedEntries: var seq[M], foreignKeyFieldName: static string) =
-  ## Fetches all entries of a "many" side from a one-to-many relationship 
-  ## between the model of `oneEntry` and the model of `relatedEntries`. It is
-  ## ensured at compile time that the field specified here is a valid foreign key
-  ## field on oneEntry pointing to the table of the `relatedEntries`-model.
+  ## Fetches all entries of a "many" side from the single one-to-many relationship 
```

Comment:
This is only a minor doc change, it felt too inaccurate after I read over it again, so I adjusted the wording a smidge.